client/jetpack-connect/controller.js

```diff
@@ -213,6 +214,13 @@ export function signupForm( context, next ) {
 	next();
 }
 
+export function credsForm( context ) {
+	removeSidebar( context );
+	context.primary = (
+		<RewindCredentialsForm role="main" siteId={ context.params.siteId } allowCancel={ false } />
+	);
+}
```

Comment:
Oh and because of Single-Tree Rendering, you need to call `next()` here (that's passed to your controller as second argument).

Reply:
true! removed it already and it solved it 👍 Was waiting for an excuse in getting up to speed with the SST news :)